packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -217,11 +225,12 @@ export enum ProcessStatus {
     COMPLETED = 'completed',
 }
 
-export interface ProcessDetailStep {
+export interface Step {
```

Comment:
I would say that the previous name was better, as it is more descriptive, ProcessStep might also be nice

Reply:
I can be a task step asweel thought right?

Reply:
True, but tasks are still Processes and are typed like that